akka-actor/src/main/resources/reference.conf

```diff
@@ -109,6 +109,10 @@ akka {
     # this is only intended for testing.
     serialize-messages = off
 
+    # Additional serialization bindings which are enabled automatically when allow-java-serialization is disabled.
+    # settings are provided
+    java-serialization-disabled-additional-serialization-bindings = ${?akka.actor.java-serialization-disabled-additional-serialization-bindings}
```

Comment:
what is this?
why is this not simply an empty section?
```
java-serialization-disabled-additional-serialization-bindings {
}
```

Reply:
I think this works the same here as it does with lists, such that other reference.conf can add values to this list if they wanted to. Is this not needed with maps?

Reply:
no, maps are merged, that's how we do it with the ordinary serialization config sections

Reply:
Ah right :) Excellent, removing the ?ref